hash/megadriv.xml

```diff
@@ -16970,6 +16992,17 @@ but dumps still have to be confirmed.
 		</part>
 	</software>
 
+	<software name="gravibots">
+		<description>Gravibots</description>
+		<year>2021</year>
+		<publisher>Mega Cat Studios</publisher>
+		<part name="cart" interface="megadriv_cart">
+			<dataarea name="rom" width="16" endianness="big" size="1441792">
+				<rom name="gravibots.bin" size="1441792" crc="0d66d022" sha1="71fa92a1d3cb61c1dc71a80e8029f98a988ba2f8"/>
+			</dataarea>
+		</part>
+	</software>
```

Comment:
I had a quick look at their page for this game, and it didn’t seem to have a download link, or at least not an obvious one.  Have they actually released this for free?  We generally don’t add original software to the software lists if it’s less than three years and hasn’t been released for free by the publisher.  (“Pay what you want” where zero is an acceptable amount but you need to confirm you’re paying zero to download doesn’t count as “released for free” for this purpose.)

Reply:
They've explicitly written "Grab a ROM for FREE or PWYW" but only on the 2019 ones. I assume that's their general stance, but I can't be sure. I'll redo this PR I guess, when I get a chance.